ns3ai Installation

Hi All,

I'm having troubles running the ./ns3 configure after installing ns3-ai into the contrib folder. I get the following error:

    CMake Error at scratch/CMakeLists.txt:54 (target_link_libraries):
    Target "scratch_scratch-simulator" links to:
    Boost::program_options
    
    but the target was not found.  Possible reasons include:
    
    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
    
    Call Stack (most recent call first):
    scratch/CMakeLists.txt:67 (create_scratch)

I'm not sure how to debug this. I think I have installed the requirements correctly as boost and protobuf appear as a library in the /usr/lib directory.

A few things to note:

- I am running arch linux (Specifically Manjaro), but I dont see how that would make a difference if all the packages are install correctly?
- protobuf_generate function is also not found - it says its using a local copy, but, as said previously, I have verified that the libraries exist.


Any help regarding my problem is appreciated!

Many thanks,
C

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,7 +76,7 @@ build_lib(
         LIBNAME ai
         SOURCE_FILES ${msg_interface_srcs} ${gym_interface_srcs}
         HEADER_FILES ${msg_interface_hdrs} ${gym_interface_hdrs}
-        LIBRARIES_TO_LINK ${libcore} Boost::program_options protobuf::libprotobuf
+        LIBRARIES_TO_LINK ${libcore} protobuf
 )
 
 # protobuf_generate function is missing in some installations by package manager

